components/script/dom/htmltimeelement.rs

```diff
@@ -44,10 +43,7 @@ impl HTMLTimeElementMethods for HTMLTimeElement {
         if element.has_attribute(&local_name!("datetime")) {
             return element.get_string_attribute(&local_name!("datetime"))
         } else {
-            match element.GetInnerHTML() {
-                Ok(x) => x,
-                _ => DOMString::new(),
-            }
+            element.upcast::<Node>().child_text_content()
```

Comment:
Oh, I was reading the wrong section. I was looking at https://html.spec.whatwg.org/multipage/semantics.html#datetime-value.